lib/galaxy/tool_util/xsd/galaxy.xsd

```diff
@@ -1457,6 +1457,20 @@ discouraged, to use the value specified as `truevalue` or `falsevalue`.
 Data input parameters can be given as a file name. The file should exist in the
 `test-data` folder. Multiple files can be specified as comma separated list.
 
+Note that it is possible to use zipped (gz, bz2) files in the `test-data` folder
```

Comment:
That is a detail that depends on the Galaxy config. Compressed files in a git repository aren't a good idea, thr git blobs themselves are already compressed. I would not recommend this.

Reply:
> That is a detail that depends on the Galaxy config.

Which part of the config do you mean?

> Compressed files in a git repository aren't a good idea, thr git blobs themselves are already compressed. I would not recommend this.

Isn't it still smaller? Anyway, here https://github.com/galaxyproject/tools-iuc/pull/4384 this was used to use a singe test file for tests with compressed data and uncompressed data (so its not only the compression that saves space but also that the uncompressed file for other tests does not need to be saved).

But I also thought about the problem that the diff between compressed files in a git context is really not helpful. So this should only be used if the test data is not supposed to change (which is the usual case for inputs).

Reply:
> Which part of the config do you mean?
> 
> >

`auto_decompress`, that's an API option at this point, but I am not sure we will not ever change the default.

Reply:
>  but I am not sure we will not ever change the default.

Now that its used in IUC tool tests it seems to be a good time point to decide on this, or?

Reply:
Then, how about disabling this? Maybe at least for the tests? If you agree: Could you point me to the relevant part of the code?

Reply:
I would be extremely said if `auto_decompress` would ever be disabled.
It's very useful in trainings if you need a small input data transfer from a remote source, but the tools used in the training cannot work with the compressed format.
I'm using this in https://training.galaxyproject.org/training-material/topics/variant-analysis/tutorials/somatic-variants/tutorial.html#variant-annotation-and-reporting, for example to import a vcf.gz file as vcf.

Reply:
> I'm using this in https://training.galaxyproject.org/training-material/topics/variant-analysis/tutorials/somatic-variants/tutorial.html#variant-annotation-and-reporting, for example to import a vcf.gz file as vcf.

But your files on disk are vcf.gz ... this is just wrong independently of the auto_decompress option.